worker/src/RTC/Router.cpp

```diff
@@ -2,6 +2,7 @@
 // #define MS_LOG_DEV_LEVEL 3
 
 #include "RTC/Router.hpp"
+#include "DepLibUring.hpp"
```

Comment:
So if we check the macro we must do it everywhere. We do it in Worker.cpp so must also be done here:

```suggestion
#ifdef MS_LIBURING_SUPPORTED
#include "DepLibUring.hpp"
#endif
```